added license details and Windows installation steps to README.md (#41)
ArnoldSmith86/virtualtabletop: README.md

This project started after [a reddit post](https://www.reddit.com/r/PlayingCardsIO/comments/jiajth/mildly_ot_feature_requests_for_a_pcio_clone/) in our [playingcardsio subreddit](https://www.reddit.com/r/PlayingCardsIO/).

## License

The project is licensed under [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html).

The cards in [assets/cards-default](https://github.com/ArnoldSmith86/virtualtabletop/tree/main/assets/cards-default) were [created by Adrian Kennard](https://www.me.uk/cards/) and released under CC0 Public Domain license.

The cards in [assets/cards-plastic](https://github.com/ArnoldSmith86/virtualtabletop/tree/main/assets/cards-plastic) have their own [license.txt file](https://github.com/ArnoldSmith86/virtualtabletop/blob/main/assets/cards-plastic/license.txt).

> 

Reviewer: Should attribution for the files under assets/icons-white be made as well?

```suggestion

The icons in [assets/icons-white](https://github.com/ArnoldSmith86/virtualtabletop/tree/main/assets/icons-white) were found at fontawesome.com with license [CC BY 4.0](https://fontawesome.com/license/free).
```

npm install                                                                             # uses the Node.js package manager to install all dependencies
```

> 

Reviewer: Since you're adding Windows, here's one for mac:

```suggestion

### MacOS
Using [brew](https://brew.sh/):

    brew install node
    git clone --recurse-submodules https://github.com/ArnoldSmith86/virtualtabletop.git
    cd virtualtabletop
    npm install

```

This will serve the project at [localhost:8272](http://localhost:8272).

> If you close you terminal and wants to restart you server you will need to get back to you clone folder using cd

Reviewer: ```suggestion
If you close your terminal and want to restart your server, you will need to first get back to your `virtualtabletop` folder using `cd`.
```
